.github/workflows/pr-notification.yml

```diff
@@ -0,0 +1,55 @@
+name: PR Notification to Google Chat
+
+on:
+  pull_request:
+    types: [opened]
+    branches:
+      - develop
+
+jobs:
+  notify:
+    runs-on: ubuntu-latest
+    if: github.event.pull_request.base.ref == 'develop'
```

Comment:
if: github.repository == 'AOT-Technologies/forms-flow-ai' add this and no need check base.ref because already branch is there in branches

Reply:
add synchronized in types